www/src/components/thumbprint-atomic/table/index.jsx

```diff
@@ -10,45 +10,53 @@ const cleanSelector = selector =>
         .replace(/hover.*:focus/, '')
         .replace(/.*> \*/, '');
 
-const Table = ({ atomicClasses }) => (
-    <table className="w-100 tp-body-2">
-        <tbody>
-            {atomicClasses.map(classItem => (
-                <tr key={classItem.selectors.join('')} className="bb b-gray-300">
-                    <td className="v-top w-40 pv2">
-                        <table>
-                            <tbody>
-                                {classItem.selectors.map(v => (
-                                    <tr key={v}>
-                                        <td data-algolia="include">
-                                            <InlineCode theme="plain" shouldCopyToClipboard>
-                                                {cleanSelector(v)}
-                                            </InlineCode>
-                                        </td>
-                                    </tr>
-                                ))}
-                            </tbody>
-                        </table>
-                    </td>
-                    <td className="v-top w-60 pv2 black-300">
-                        <table>
-                            <tbody>
-                                {classItem.declarations.map(v => (
-                                    <tr key={v}>
-                                        <td>
-                                            <InlineCode theme="plain">{v}</InlineCode>
-                                            <Swatch value={v} />
-                                        </td>
-                                    </tr>
-                                ))}
-                            </tbody>
-                        </table>
-                    </td>
-                </tr>
-            ))}
-        </tbody>
-    </table>
-);
+const Table = ({ atomicClasses }) => {
+    const atomicClassesHackless = atomicClasses.filter(
+        // There are `shadow-*, _:-ms-lang(x)` hack selectors in the Atomic source code that
+        // provide darker box-shadows for IE/Edge. This filters out those selectors to prevent them
+        // from rendering in these docs.
+        item => item.selectors[1] !== '_:-ms-lang(x)',
```

Comment:
Probably a little better to do `item => !item.selectors.includes('_:-ms-lang(x)')`.

Reply:
Yeah, better. Just updated.